client/src/partials/enterprises/enterprises.html

```diff
@@ -20,235 +20,181 @@
-          </div>
-          <table class="table table-condensed table-hover">
-            <thead>
-              <tr>
-                <th>{{ 'TABLE.COLUMNS.NAME' | translate }}</th>
-                <th>{{ 'TABLE.COLUMNS.ABBREVIATION' | translate }}</th>
-                <th>{{ 'TABLE.COLUMNS.PO_BOX' | translate }}</th>
-                <th>{{ 'TABLE.COLUMNS.EMAIL' | translate }}</th>
-                <th>{{ 'TABLE.COLUMNS.PHONE' | translate }}</th>
-                <th>{{ 'TABLE.COLUMNS.EDIT' | translate }}</th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr ng-repeat="e in EnterpriseCtrl.enterprises | orderBy:'name' track by e.id">
-                <td>{{ e.name }}</td>
-                <td>{{ e.abbr }}</td>
-                <td>{{ e.po_box }}</td>
-                <td>{{ e.email }}</td>
-                <td>{{ e.phone }}</td>
-                <td>
-                  <button type="button" id="enterprise-{{e.id}}" ng-click="EnterpriseCtrl.update(e)" class="btn btn-xs btn-default">
-                    <span class="fa fa-pencil"></span>
-                  </button>
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </div>
-      </div>
+      <!-- enterprise form  -->
+      <div class="col-xs-12 col-sm-6">
+        <form
+          name="EnterpriseForm"
+          bh-submit="EnterpriseCtrl.submit(EnterpriseForm)"
+          bh-form-defaults
+          novalidate>
 
-      <!-- Optional Information Form -->
-      <div class="col-md-6" ng-switch="EnterpriseCtrl.view">
-        <div ng-switch-default>
-          <div class="alert alert-info">
-            <h4>{{ 'ENTERPRISE.TITLE' | translate }}</h4>
-            <p>{{ 'ENTERPRISE.DESCRIPTION_1' | translate }}
-            </p>
-            <ul class="list-plain">
-              <li><span class="fa fa-pencil"></span> {{ 'ENTERPRISE.DESCRIPTION_2' | translate }}</li>
-              <li><span class="fa fa-plus-circle"></span> {{ 'ENTERPRISE.DESCRIPTION_3' | translate }}</li>
-            </ul>
-          </div>
-        </div>
+          <!-- enterprise main info  -->
+          <div class="panel panel-primary">
 
-        <div class="panel panel-primary" ng-switch-when="create">
-          <div class="panel-heading">
-            {{ "ENTERPRISE.CONFIGURATION_ENTERPRISE" | translate }}
-          </div>
-          <div class="panel-body">
-            <form
-              name="CreateForm"
-              bh-submit="EnterpriseCtrl.submit(CreateForm)"
-              bh-form-defaults
-              novalidate>
-
-              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.name.$invalid }">
+            <div class="panel-heading">
+              <b><i class="fa fa-building"></i> {{ 'TREE.ENTERPRISE' | translate }} :
+                <span ng-if="EnterpriseCtrl.enterprise.name">{{ EnterpriseCtrl.enterprise.name | uppercase }}</span>
```

Comment:
This really should be changed.  The web model is usually:
 1. HTML for content/layout
 2. CSS for styling/transformations
 3. Javascript for interactivity

This code doesn't make sense with that model, and I do not see a reason why it couldn't be done with CSS instead.

Finally, every time you write `{{}}` in angular, the browser has to remember to check binding for every single change on the page.  When you have a filter, it will re-compute the filter.  So, for every single action in this module, you will be computing `uppercase(EnterpriseCtrl.enterprise.name)`.  With CSS, the browser will never have to work again - only if `EnterpriseCtrl.enterprise.name` changes.

[Here is a link](http://getbootstrap.com/css/#type-transformation) to the bootstrap classes to make text uppercase.